pkg/keystore/keystore.go

```diff
@@ -134,7 +137,10 @@ func (ks *Keystore) DeleteTrustedKeyRoot(fingerprint string) error {
 // MaskTrustedKeySystemRoot masks the system root trusted key identified by fingerprint.
 func (ks *Keystore) MaskTrustedKeySystemRoot(fingerprint string) (string, error) {
 	dst := path.Join(ks.LocalRootPath, fingerprint)
-	return dst, ioutil.WriteFile(dst, []byte(""), 0644)
+	if err := ioutil.WriteFile(dst, []byte(""), 0644); err != nil {
+		return dst, err
+	}
+	return dst, os.Chmod(dst, 0644)
```

Comment:
Minor nit: I would prefer the err-handling for chmod on its own, and `return dst, nil` at the end (here and in the similar code above).

Reply:
ok I'll change that.